doc/nrf/gs_modifying.rst

```diff
@@ -150,9 +231,30 @@ See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how
 To locate a specific configuration option, use the filter (:guilabel:`Jump to` in menuconfig and guiconfig).
 The documentation for each :ref:`configuration option <configuration_options>` also lists the menu path where the option can be found.
 
-.. important::
-   All changes to the :file:`.config` file are lost when you clean your build directory.
-   You can save it to another location, but you must then manually copy it back to your build directory.
+Changing the configuration permanently
+======================================
+
+To configure your application and maintain the configuration when you clean the build directory pristinely, you need to specify the configuration in one of the permanent configuration files.
+In most cases, this will be either the default :file:`prj.conf` file of the application or an extra Kconfig fragment.
```

Comment:
`In most cases` ?
If user is not doing this in `prj.conf` or in an extra Kconfig fragment, then what options are left ?.

Suggest to remove `In most cases`.
```suggestion
This can be the default :file:`prj.conf` file of the application or an extra Kconfig fragment.
```